sum/sum.py

```diff
@@ -0,0 +1,17 @@
+WORDS = ["a", "neat", "beat", "house", "me", "button", "mat"]
+
+
+def tuple_of_word_lengths():
+    """Return a tuple with the length of the shortest word,
+    the length of the longest word, and the average word length."""
+    word_lengths = ()
+
+    longest_word = max(WORDS, key=lambda x: len(x))
```

Comment:
Nice use of lambdas here! To make it even simpler, `key` can take in a custom lambda like you did here, or even a function name, like `len`, that it uses to base it's evaluation on.

https://docs.python.org/3.7/library/functions.html#max

So I believe this would work:

```
longest_word = max(WORDS, key=len)
```